include/z64.h

```diff
@@ -681,15 +681,15 @@ enum fram_command {
     FRAM_COMMAND_SET_MODE_READ_AND_STATUS = 0xF0000000,
     /* unk */
     FRAM_COMMAND_UNK_ERASE_OPERATION = 0x3C000000
-};
-
-enum fram_mode {
-    FRAM_MODE_NOP = 0,
-    FRAM_MODE_ERASE,
-    FRAM_MODE_WRITE,
-    FRAM_MODE_READ,
-    FRAM_MODE_STATUS
-};
+} fram_command;
```

Comment:
```suggestion
} FramCommand;
```
To match current style.